intellij/workspace/update/idea.properties

```
@@ -2,4 +2,5 @@ idea.config.path=$[WORKSPACE_PATH]/.intellij/config
 idea.system.path=$[WORKSPACE_PATH]/.intellij/system
 idea.plugins.path=$[DEVON_IDE_HOME]/plugins/intellij
 idea.log.path=$[WORKSPACE_PATH]/.intellij/system/log
+idea.log.console=$[IDE_ENABLE_LOG]
```

Please adjust to: https://github.com/devonfw/IDEasy/pull/729#issuecomment-2457123159
```suggestion
idea.log.console=$[IDE_ENABLE_CONSOLE_LOG]
```